bigbone/src/main/kotlin/social/bigbone/api/entity/Announcement.kt

```diff
@@ -0,0 +1,148 @@
+package social.bigbone.api.entity
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents an announcement set by an administrator.
+ * @see <a href="https://docs.joinmastodon.org/entities/Announcement/"> Mastodon API Announcement </a>
+ */
+
+@Serializable
+data class Announcement(
+    /**
+     * The ID of the announcement in the database.
+     */
+    @SerialName("id")
+    val id: String = "0",
+
+    /**
+     * The text of the announcement.
+     */
+    @SerialName("content")
+    val content: String = "",
+
+    /**
+     * When the announcement will start.
+     */
+    @SerialName("starts_at")
+    val startsAt: String? = null,
```

Comment:
Let’s update this to match the recent changes implemented in #312:

```suggestion
    @Serializable(with = DateTimeSerializer::class)
    val startsAt: PrecisionDateTime = InvalidPrecisionDateTime.Unavailable,
```

Reply:
Sorry! I guess I forgot to change this again when I was moving from wsl to windows